multicluster/controllers/multicluster/core/resourceimport_controller.go

```diff
@@ -143,15 +145,27 @@ func (r *ResourceImportReconciler) handleResImpUpdateForService(ctx context.Cont
 	if err != nil && !apierrors.IsNotFound(err) {
 		return ctrl.Result{}, err
 	}
-	var creationSucceed bool
-	mcsObj := getMCSForServiceImport(resImp)
+	if !reflect.DeepEqual(*svc, corev1.Service{}) {
```

Comment:
Is it to check if the Service exists or not? If so should we just check IsNotFound, or add a bool variable for that (which can replace svcCreated, svcImpCreated).

Reply:
done via using `svcImpNotFound`